base/ltmiscen.dtx

```diff
@@ -276,6 +276,20 @@
 % \changes{v1.0w}{1994/11/30}{(DPC) Use \cs{@dofilelist}}
 % \changes{v1.2a}{2022/11/06}{Repeat release info at the end (gh/944)}
 %    \begin{macrocode}
+%    \end{macrocode}
+%    If we roll back we have to drop stuff before adding chunks,
+%    otherwise the code will just be appended, i.e., doubled.
+%    This would result in a harmless warning during normal format
+%    built, because in that case the chunk label doesn't exist.
```

Comment:
I cannot follow this well enough to correct it!
Bad bits are: 
"normal format built" (all 3 words!)
"chunk label" (maybe this makes sense with more context?)

Reply:
the code added to a hook are called code chunks throughout the documentation and they are labeled. Anyay, rewriten